common/building_groups/00_building_groups.txt

```diff
@@ -93,6 +93,15 @@ bg_munition_plants = {
 	infrastructure_usage_per_level = 3
 }
 
+bg_charcoal_kiln = {
+	parent_group = bg_light_industry
+
+	urbanization = 5
+	infrastructure_usage_per_level = 1
```

Comment:
10 urbanization and 2 infrastructure usage would be more consistent with the building costs 400 (similar to a mine)